services/crypto_update.proto

```diff
@@ -171,8 +171,13 @@ message CryptoUpdateTransactionBody {
         VirtualAddress add = 20;
 
         /**
-         * The 20-byte EVM address of the virtual address that is being removed.
-         */
-        bytes remove = 21;
+        * The 20-byte EVM address of the virtual address that is being disabled and therefore removed from active use on the account.
+        */
+        bytes disable = 21;
```

Comment:
That's fair.
We'll coordinate the disable addition.
Will remove as suggested since adding to a oneof later on (even if the ordinal value is greater than existing outer fields) won't break the API